docs/user_guide/algorithm_list.rst

```diff
@@ -139,6 +163,49 @@ Graphs are often too large to handle, so a portion of the graph is extracted. Of
     This algorithm finds a maximal subgraph that contains nodes of at least degree *k*.
 
 
+.. py:function:: subgraph.k_truss(graph: Graph(is_directed=False), k: int) -> Graph
+
+    Finds the subgraph of edges with are part of k-2 triangles.
```

Comment:
I think it would be slightly more clear to state the  it finds the "maximal induced subgraph".

I think there's a typo where "with" should be replaced with "which".